install/install_for_macos_linux.sh

```diff
@@ -32,24 +53,38 @@ if [ -f "/etc/arch-release" ]; then
   pacman -S --needed --noconfirm openssh
 fi
 
+should_install_unzip() {
+  if [[ $(command -v unzip) ]]; then
+    return 1
+  fi
+}
+
+CURRENT_USER=$(whoami)
+SUDO_COMMAND=''
+if  command -v sudo &> /dev/null
+then
+    SUDO_COMMAND='sudo'
+fi
```

Comment:
When `sudo` isn't available, you could fall back to using [`doas`](https://wiki.debian.org/Doas) instead

Reply:
Cool tip do you know if it’s available on Arch per default?

Reply:
Thank you very much for pointing this tool.

Reply:
> Cool tip do you know if it’s available on Arch per default?

I don't know for sure, but I think it isn't. It's more popular in the BSD world, but is also used on Linux by people (like me) who prefer a smaller and safer alternative.

I would set `SUDO_COMMAND` to `sudo` if available, then fallback to `doas` if available, and then warn that no tool to elevate privileges was found. Oh, by the way, a lot of distros ship `pkexec` too, as part of the Polkit (and systemd) suite